src/taco-tests-utils/tacoErrorTestHelper.ts

```diff
@@ -38,4 +34,20 @@ export class TacoErrorTestHelper {
             }
         });
     }
+
+    public static verifyExcludedTacoErrors(fileName: string, resources: any, excludedErrorCode: number): void {
+        var errorCodes: any = TacoErrorTestHelper.getErrorCodes(fileName);
+        var excludedErrorCodeStr: string = errorCodes[excludedErrorCode];
+        shouldModule(excludedErrorCodeStr).equal(undefined, "Exclude error code " + excludedErrorCode + "shouldn't be present");
```

Comment:
It might be slightly clearer to do `shouldModule(excludedErrorCode in errorCodes).be.false` to make sure that excludedErrorCode isn't a key, and not just a key mapping to undefined? Shouldn't really matter in our case though.